setup.py

```diff
@@ -45,6 +45,8 @@
         "pyxb==1.2.6",
         "ismrmrd==1.9.1",
         "tensorboard>=2.5.0",
+        "openvino-dev==2021.4.2",
+        "openvino-extensions==2021.4.2",
```

Comment:
Seems like there is a dependency conflict. Try different version of numpy (`numpy<1.20.0`)